Update installation instructions for MacOS (#11957)
PrairieLearn/PrairieLearn: docs/dev-guide/quickstart.md

> # Quickstart

Reviewer: Nit: should we label this "Development quickstart", or something to make it clear that this is development only? Someone searching for "quickstart" might find this and get scared off by this being all technical. Who knows if that'd happen in practice though 🤷

Author: I think that this would tie in nicely with the content reorganization of top-level headers, which would make it very clear what section of the docs you are viewing. Changed in 534c154ac9555dfbd627d9c6125483e09d0cc8de.

```sh
sudo make dev-workspace-host # or sudo make start-workspace-host
```

## Documentation

If you want to preview the documentation, run:

```sh
make preview-docs
```

## Testing

If you are contributing code to PrairieLearn, you must ensure that your changes work and pass our style guidelines. More information on debugging and testing can be found on the [developer guide](./guide.md). Information on contributing can be found on the [contribution guide](../contributing.md).

Run the test suite (Docker must be installed and running):

```sh
make test
```

Or, to run tests for just a specific language:

```sh
make test-js     # Javascript only
make test-python # Python only
```

### JavaScript tests

The main JavaScript test suite can take on the order of 10+ minutes to run, so you may only run specific test files as you develop. To run specific test files, you first need to run `make start-support` to start the database and other services, and then you can run specific test files.

```sh
make start-support
cd apps/prairielearn
yarn mocha src/tests/getHomepage.test.ts
```

### Package tests

> To test specific packages, you can run it from the package directory you would like to test.

Reviewer: Nit:

```suggestion
To test a specific package, you can run `yarn test` from the package's directory.
```
